internal/mempool/mempool.go

```diff
@@ -1727,12 +1727,12 @@ func (mp *TxPool) pruneStakeTx(requiredStakeDifficulty, height int64) {
 		}
 	}
 	for _, tx := range mp.staged {
-		txType := stake.DetermineTxType(tx.MsgTx())
+		txType := tx.Type
 		if txType == stake.TxTypeSStx &&
-			tx.MsgTx().TxOut[0].Value < requiredStakeDifficulty {
+			tx.Tx.MsgTx().TxOut[0].Value < requiredStakeDifficulty {
```

Comment:
I think renaming the `TxDesc` var to `desc` here would make this read better. We'd have `desc.Tx` instead of `tx.Tx`.

Reply:
Was aiming to match the other code in this method, but I agree. Both loops updated.